useful-scripts/publisher-tests/index.ts

```diff
@@ -0,0 +1,340 @@
+/* eslint-disable */
```

Comment:
Many of the comments I've left for the importer-tests file also apply to this file.